docs/howtos/extending/event_loop.md

```diff
@@ -1,20 +1,47 @@
 (intro-to-event-loop)=
 
-# An introduction to the event loop in napari
```

Comment:
Not sure if this doc fits in this section? It's not really a guide to extending napari at all. I would maybe keep this under `in depth explanations` and link to it from the top of the `Hooking up your own events` doc?

Reply:
Yeah good point, will amend